data/brands/amenity/post_box.json

```diff
@@ -1,6 +1,6 @@
 {
   "properties": {
-    "path": "operators/amenity/post_box",
+    "path": "brands/amenity/post_box",
```

Comment:
To be clear, https://github.com/osmlab/name-suggestion-index/issues/5561#issuecomment-954028769 described [an edit to the `amenity=post_office` documentation](https://wiki.openstreetmap.org/wiki/Special:Diff/2213092) that should’ve had little bearing on `amenity=post_box`. Conversely, these features were originally moved from the `brands` tree to the `operators` tree in response to feedback in #4925, but that feedback was about `amenity=post_box`, not `amenity=post_office`.

I think the back and forth is because post offices and postboxes are being treated identically. But it seems like the expectation – most of the time – is that post offices need the distinction between brands and operators more than postboxes do. Contract post offices are a widespread phenomenon in many countries, but contract postboxes seem would be much more of an edge case.

(I did argue in gravitystorm/openstreetmap-carto#1023 that postboxes should be labeled based on their `operator`s, but that was before I learned about `brand` and before it had become common on postal facilities in the U.S.)

Reply:
oh ok - If I'm reading your comment correctly, it sounds like `amenity/post_box` should remain in `operators` tree?  That seems ok to me.